cmd/telemetrygen/internal/common/config.go

```diff
@@ -92,4 +104,8 @@ func (c *Config) CommonFlags(fs *pflag.FlagSet) {
 	fs.Var(&c.ResourceAttributes, "otlp-attributes", "Custom resource attributes to use. The value is expected in the format key=\"value\"."+
 		"Note you may need to escape the quotes when using the tool from a cli."+
 		"Flag may be repeated to set multiple attributes (e.g -otlp-attributes key1=\"value1\" -otlp-attributes key2=\"value2\")")
+
+	c.TelemetryAttributes = make(map[string]string)
+	fs.Var(&c.TelemetryAttributes, "telemetry-attributes", "Custom telemetry attributes to use. The value is expected in the format \"key=\\\"value\\\"\". "+
+		"Flag may be repeated to set multiple attributes (e.g --signal-attributes \"key1=\\\"value1\\\"\" --otlp-attributes \"key2=\\\"value2\\\"\")")
```

Comment:
`signal-attributes` ? Did you mean `otlp-attributes`?

Reply:
sry, there was a typo. Both should be signal-attributes. This flag is to set telemetry signal attributes